templates/pages/dial.html

```diff
@@ -127,6 +127,11 @@ <h1>SFS Switchboard</h1>
 
       const response = await fetch(`${formAction}.json?${params}`);
       const result = await response.json();
+
+      if (!result || !result.rows) {
+        throw exception("There was an error querying the database")
```

Comment:
Needs to be `Exception`

Reply:
I don't think so. It works correctly this way, and notice a few lines down:
if (!recordLink)
          throw exception("No link for record", result);

Reply:
Ah, in here we have this function:
function exception(message, ...context) 
I'm not passing in result as the context since it might be null, but I could if that makes usage clearer.

Reply:
Ah, you are correct; I'd forgotten about the `exception()` function. Apologies for the drive-by comment.